### DISCUSSION TITLE
We are low on first-timer issues

### DISCUSSION THREAD

**Discussion by damithc** [MarkBind/markbind#1629] [Dev team discussions]
@MarkBind/active-devs our list of good first issues is getting quite short https://github.com/MarkBind/markbind/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22

**Comment by ang-zeyu** [comment]
Hijacking this, sorry 😅

@MarkBind/active-devs I've documented a bunch of new issues for this semester, although some of them are more suitable for 3282. In particular I would like to highlight this issue https://github.com/MarkBind/markbind/issues/2077. Feel free to tackle them if you're interested.

If you need more tasks, feel free to let me, @jonahtanjz or @damithc know as well. I can try to spend some more brain juice on coming up with tickets.

Of course, it would be even better if you come up with ideas to work on yourself, and the requirements for them, especially substantial ones. Devops ideas, documentation nits are important as well but as the project is still small, product ideas would be even more valuable. There is also still a lot of technical debt, any tickets on those are also very valuable.
